### DISCUSSION TITLE
[QUESTION] longhorn ext4 volume jbd2 has a lot IO,Does turning off the log function (has_journal) affect the volume of longhorn?

### DISCUSSION THREAD

**Discussion by adababys** [longhorn/longhorn#6323] [General]
## Question
> Suggest to use https://github.com/longhorn/longhorn/discussions to ask questions.
![longhorn-volume-jbd2-high-io-1](https://github.com/longhorn/longhorn/assets/8872412/3cc5c501-e228-464c-8d87-ebe12f4dcb20)
![longhorn-volume-jbd2-high-io-2](https://github.com/longhorn/longhorn/assets/8872412/a46acec2-28e0-4832-978e-bc83bfcd1800)

![has_journal](https://github.com/longhorn/longhorn/assets/8872412/835030b1-36ce-48cb-836d-f99b7538173e)

## Environment

 - Longhorn version:1.4.1
 - Kubernetes version:1.23.13
 - Node config
   - OS type and version: openEuler
   - CPU per node:
   - Memory per node:
   - Disk type
   - Network bandwidth and latency between the nodes:
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal):Baremetal

**Comment by derekbit** [comment]
It really depends on your application and filesystem resilience.
